web/src/pages/HomePage/HomePage.tsx

```diff
@@ -22,7 +22,7 @@ const HomePage = () => {
           </Link>
 
           <Link
-            className="relative block h-full w-full rounded bg-[url('/images/FindImage.webp')] bg-cover text-lg text-white shadow hover:shadow-white"
+            className="relative block h-full w-full rounded bg-[linear-gradient(to_bottom_right,rgba(0,0,0,0.51),rgba(0,0,0,.1)),url('/images/FindImage.webp')] bg-cover text-lg text-white shadow hover:shadow-white"
```

Comment:
here would be ( the .3 can be changed to whatever looks best, just want the other side to be 0)

`to_bottom_right,rgba(0,0,0,0),rgba(0,0,0,.3)`